nilearn/experimental/surface/_maskers.py

```diff
@@ -0,0 +1,136 @@
+from __future__ import annotations
+
+from typing import Any
+
+import numpy as np
+
+from nilearn.experimental.surface._surface_image import SurfaceImage
+
+
+class SurfaceMasker:
+    # TODO:
+    # - allow passing an actual roi mask
+    # - don't store a full array of 1 to mean "no masking"
+    # - 1d output for 1d images
+
+    mask_img_: SurfaceImage | None
+    output_dimension_: int | None
+
+    def _fit_mask_img(self, img: SurfaceImage) -> None:
```

Comment:
I don't understand what to expect from this method. A short docstring could help.